### ISSUE TITLE
How to check a PyObject for None in C#?

### ISSUE THREAD

**Issue by henon** [pythonnet/pythonnet#869]
How to check an object returned from Python against None?

I can't find a good way to do it. This is my workaround which is very unsatisfying. 

```csharp
                var type = pyobj.__class__.ToString();
                if (type== "<class 'NoneType'>")
                    return null;
```

A simple `pyobj.IsNone` would be cool.  What am I missing?

**Comment by henon** [comment]
Ok, I found it: `Runtime.CheckNone()`

**Comment by filmor** [comment]
If you make this a property (i.e. `public bool IsNone => CheckNone(this) == null`) and add a simple test, I'll merge the change :)

**Comment by henon** [comment]
There are also IsTrue() and IsFalse() methods in PyObject. Shouldn't it be consistent with those?

**Comment by filmor** [comment]
Hah, good point. Yeah, it probably should :)

**Comment by filmor** [comment]
Fixed in #1137.
